webpack.common.js

```diff
@@ -16,46 +15,46 @@ module.exports = {
         'sva-finder': './src/Components/SvaFinder.js',
         'act-widget': './src/Components/ActWidget.js',
         'donation-widget': './src/Components/DonationWidget.js',
-        pdfworker: './src/Utility/PdfWorker.js',
         'test-interface': './src/test-interface.js',
         // We need to define a dummy entrypoint that requires all our translation files, otherwise Webpack will not
         // process them.
         'translations-dummy': ['./src/i18n/de.json', './src/i18n/en.json', './src/i18n/fr.json', './src/i18n/pt.json'],
     },
     optimization: {
-        minimize: true,
-        minimizer: [
-            new TerserPlugin({
-                sourceMap: true,
-                extractComments: false,
-                cache: true,
-                parallel: true,
-                terserOptions: {
-                    mangle: {
-                        reserved: [
-                            'ActionButtonPlaceholder',
-                            'NewRequestButtonPlaceholder',
-                            'CompanySelectorPlaceholder',
-                            'RequestFormPlaceholder',
-                            'DynamicInputContainerPlaceholder',
-                            'SignatureInputPlaceholder',
-                            'RequestTypeChooserPlaceholder',
-                            'RecipientInputPlaceholder',
-                            'TransportMediumChooserPlaceholder',
-                        ],
-                    },
+        splitChunks: {
```

Comment:
Not sure if this configuration is a good fit for us, to be honest. While I agree that duplicating code across the different entrypoints isn't great, we are trying pretty hard to reduce the code that needs to be downloaded per page. We have a few pages (mostly those with a generator embedded) that are *really* heavy but most of our pages are optimized to load as little JS as possible. Currently, Webpack 'only' complains about the asset sizes in the three generator-related files:

![image](https://user-images.githubusercontent.com/4048809/98464627-1b44e400-21bc-11eb-8119-b58f2b9c29dd.png)

With this change, *every* entrypoint is now a problem:

![image](https://user-images.githubusercontent.com/4048809/98464633-2a2b9680-21bc-11eb-9c1c-f113c3159158.png)

I am especially worried about this because of our blog and company single pages. They are pretty optimized to load as quickly as possible and adding that much (unnecessary) JS to them
would potentially hurt our SEO, which we need to avoid.

Reply:
Didn't check the build output, this was definitely not the intention